lib/processLayerStyles.js

```diff
@@ -1,17 +1,25 @@
-const { addOptionalProperty, colorToRGBA, doRound } = require('./processShared')
+const { addOptionalProperty, colorToRGBA, doRound, prepareAssetBundleForPDF, prepareAssetBundleForPNG } = require('./processShared')
+const fs = require("fs")
 
+const FILL_TYPE = {
+  SOLID_COLOR: 0,
+  GRADIENT: 1,
+  IMAGE: 4,
+}
 
-const processLayerStyles = (layerStyles, filepath, outputFolder, modifiers) => {
+const processLayerStyles = async (layerStyles, filepath, outputFolder, modifiers) => {
     let ret = []
 
-    layerStyles.forEach((style)=>{
+    let sketchFile = await require('sketch-to-svg').SketchFile.from(filepath)
```

Comment:
inline requires are a bit untidy - can it be moved to where the other requires are kept?